### PR TITLE
Updated russian translation for malware detection content and other changes

### PR DESCRIPTION
**Pull request by NEK-RA**
Slow as ~~always~~ turtle, I'm here again :D

- new content and changes are translated
- new screenshots added and used via relative links

What do you think about including into manual info which blacklists exactly used? 
I guess 2 of 5 which are from Github repositories can be added somewhere in the end of description of malware detection feature

### CODE REVIEW

File: ru/paid_features.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Некоторые случаи в которых эта возможность применима:
+
+- Пользователь посещает известный опасный сайт (например фишинговый или мошеннический)
+- Пользователь устанавливает некое приложение или дополнение (которое оказывается шпионит) которое соединяется с опасным доменом или IP адресом
+- На устройство любым образом попадает вредонос (например шпион, вымогатель или ботнет), который соединяется с опасным доменом или IP адресом
+
+Черные списки используемые в PCAPdroid содержат списки доменов и IP адресов с плохой репутацией. Они включают сканнеры, брутфорсеров и известные источники распространения вредоносного ПО. Черные списки в основном содержат статические правила, основанные на последних данных о заражениях, а так же некоторые динамические правила, которые сгенерированы автоматически, обычно благодаря [АСООС](https://ru.wikipedia.org/wiki/Client_honeypot)(англ. [Client Honeypots](https://en.wikipedia.org/wiki/Client_honeypot)). PCAPdroid обновляет списки раз в сутки, чтобы иметь возможность обнаруживать актуальные вредоносные соединения.
```
Comment (emanuele-f) [inline ru/paid_features.md:19]: What's ACOOC? Please replace the reference to Client Honeypots with a generic Honeypots (link: https://ru.wikipedia.org/wiki/Honeypot), sorry for providing the wrong link in tg

File: ru/paid_features.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<p align="center">
+<img src="images/capture-stats-blacklist-status.png" width="250" />
+</p>
+
+Состояние черных списков отображается на экране `Показатели`:
+
+- *Обновлено списков*: указывает количество обновленных черных списков. Если с обновление какого-либо из них возникнет проблема - будет использована предыдущая версия списка.
+- *Вредоносных доменов*: сообщает о количестве уникальных правил работающих по доменам.
```
Comment (emanuele-f) [inline ru/paid_features.md:28]: This refers to https://github.com/emanuele-f/PCAPdroid/blob/master/app/src/main/res/values-ru/strings.xml#L164 . When translating please ensure not to change the meaning of the words. Here for example the english text contains "Domain rules", which refers to the term "rule", whereas the translation contains "Malicious domains", which instead removes the term "rule" and assumes "malicious". In this case is not an issue but it's a good practice to keep the meaning unchanged.

**Comment by emanuele-f** [comment]
Hi, thank you for your efforts in updating the manual translation! Please check my comments above. Also it would be better to uniform the screenshots theme to light. The [old screenshots](https://github.com/NEK-RA/PCAPdroid/blob/russian-update/ru/images/app_details.png) use the dark theme but the english manual and the other front-pages use the light theme.

File: ru/paid_features.md (hunk 1 above)
Comment (NEK-RA) [reply ru/paid_features.md]: fixed in recent commit

File: ru/paid_features.md (hunk 2 above)
Comment (NEK-RA) [reply ru/paid_features.md:28]: I understand what do you mean, but I just used in manual the terms that I pushed into localization file. 
While translating related strings I worried about screen spase usage and were not sure how to translate it as near as possible to original meaning without usage very long string values.

If you wish I can update terms in both user manual and localization file, but I guess this should be done right before new version release. Just to be sure that current manual terms are mathed to terms in current app's version.

**Comment by NEK-RA** [comment]
> Also it would be better to uniform the screenshots theme to light.

Will do it now

File: ru/paid_features.md (hunk 2 above)
Comment (emanuele-f) [reply ru/paid_features.md:28]: Ok thanks for your explanation

**Pull request merged**